libsawtooth/src/client/rest.rs

```diff
@@ -107,6 +107,41 @@ impl SawtoothClient for RestApiSawtoothClient {
             |item: Result<Block, _>| item.map(|block| block.into()),
         )))
     }
+    /// Get the state entry with the given address from the current blockchain
+    fn get_state(&self, address: String) -> Result<Option<ClientSingleState>, SawtoothClientError> {
```

Comment:
One thing you might be able to do is change the `get` function so that instead of de-serializing as `Single<T>`, you just de-serialize as `T`. So here you'd call `get::<SingleState>`, for batches you'd call `get::<Single<Batch>>`, etc.

Reply:
This worked, modified the generic `get` function and `get_state` now uses it